package com.AISubtitles.Server;

import org.junit.jupiter.api.Test;
import org.springframework.boot.test.context.SpringBootTest;

import java.sql.SQLException;

@SpringBootTest
class ServerApplicationTests {

	@Test
	void contextLoads() throws SQLException {
//		System.out.println(dataSource.getClass());
		// System.out.println(userMapper.getClass());
		// System.out.println(userMapper.findByUserEmail("1063676802@qq.com"));
		//System.out.println(userMapper.insertUser(new User(1,"1", "1")));
//
//		Connection connection = dataSource.getConnection();
//		System.out.println(connection);

//		connection.close();
	}
}
